LDAR_Sim/src/programs/site_level_method.py

```diff
@@ -109,23 +124,46 @@ def update(self, current_date: date) -> TaggingFlaggingStats:
                         detection_record.site_id
                     )
                     existing_plan.update_with_latest_survey(
-                        detection_record, self._redund_filter, self._name, date_to_check
+                        detection_record,
```

Comment:
Instead of this huge chain of if else logic based on the stationary type and a modified follow-up survey plan, can we use a continuous specific type of survey plan and split this info 2 methods called by the original update based on deployment type.

Reply:
This will require a bigger restructuring of the existing method overhaul.